src/atoms/Heading.js

```diff
@@ -35,6 +35,15 @@ const ProtoHeading = styled.h1`
 					calc(${props.marginBottomFactor} * ${headingDefaultLineHeight})
 				;
 
+				${props.maxLines && `
+					display: -webkit-box;
```

Comment:
If we don't make it backwards-compatible, won't our build systems add browser prefixes and such that is needed?

Reply:
yeah, but we don't have the ability to multiline CSS
if we're using Firefox, text will be just cropped. That's all
in Chrome, we're getting additional ...
However, I could implement JS solution, but it requires time

Reply:
CSS > JS

Reply:
Could you add this explanation to the PR description, though?

Reply:
Sure, doing it